common/state_edicts/02_promote_military_recruitment.txt

```diff
@@ -36,100 +36,121 @@ edict_promote_military_recruitment = {
 	}
 	
 	modifier = {
-		local_manpower_modifier = 0.50
-		local_manpower = 0.250
+		local_manpower_modifier = 0.33
+		#local_manpower = 0.250
```

Comment:
Isn't reducing the percent bonus ontop of removing the flat bonus too much of a nerf?

Reply:
This is vanilla value, and the edicts are still "cheaper" in a modpack.

Reply:
Vanilla values are fine in this case?

Reply:
This Edict was worth it even in vanilla. There is still editc lvls, so it gets even stronger for the same cost.